index.ts

```diff
@@ -352,7 +354,19 @@ async function init() {
     )
   }
 
-  const { features, experimentFeatures } = result
+  if (argv.bare) {
+    result.needsBareboneTemplates = true
+  } else {
+    result.needsBareboneTemplates = await unwrapPrompt(
+      confirm({
+        message: language.needsBareboneTemplates.message,
+        // TODO: default to true sometime in the future
```

Comment:
Not a must.

But when working on the many experimental features I'm planning, I can't help but think that defaulting to barebone templates would make life easier for those who want to try out these new features.